usaspending_api/database_scripts/etl/transaction_delta_view.sql

```diff
@@ -32,6 +32,11 @@ SELECT
   UTM.product_or_service_description,
   UTM.naics_code,
   UTM.naics_description,
+  CASE
+    WHEN UTM.naics_code IS NOT NULL
+      THEN CONCAT('{"code":"', UTM.naics_code, '","description":"', UTM.naics_description, '","id":"', NULL, '"}')
```

Comment:
Same as for spending_by_psc, if `id` is not needed for the response I would not add it to the agg_key.